docs/user_guide.rst

```diff
@@ -96,7 +96,7 @@ Return status of Robocop depends on number of issues reported per given severity
 Number -1 means that return status is not affected by number of issues for given message. Default values can be configured
 by ``-c/--configure`` and ``return_status:quality_gate`` param::
 
-  robocop --configure return_status:quality_gate:E=100:F=10:I=9
+  robocop --configure return_status:quality_gate:E=100:W=10:I=9
```

Comment:
Good eye! I will also suggest W=100 so it's in line with what documentation says next
```suggestion
  robocop --configure return_status:quality_gate:E=100:W=100:I=9
```